### DISCUSSION TITLE
Naming Conventions

### DISCUSSION THREAD

**Discussion by KNadwit** [stephenh/ts-proto#637] [Q&A] [answered]
Hi,

Just had a quick question about how/where the names are processed. I ran a proto file through and single word field names were preserved but field names such as "GO_NUMBER" were converted to "goNumber". 
Any help on how I could go about modifying the repo to correct for this would be appreciated! 

<img width="288" alt="Screen Shot 2022-08-03 at 1 42 59 PM" src="https://user-images.githubusercontent.com/108360666/182707907-2aa776e6-2074-4b43-904f-6a8a6d8ebbeb.png">
<img width="316" alt="Screen Shot 2022-08-03 at 1 44 14 PM" src="https://user-images.githubusercontent.com/108360666/182708168-de0a17e8-6ceb-45a3-bc9e-c6ef722cadaa.png">

**Comment by stephenh** [comment, marked as answer]
Hi @KNadwit , the code for this is here:

https://github.com/stephenh/ts-proto/blob/main/src/case.ts#L3

But maybe you want to just set `snakeToCamel` to `false`? (See the readme for the `--ts_proto_opt` flag to `protoc`)

**Reply by KNadwit** [reply]
That was it, thank you!
